.changeset/short-emus-move.md

```diff
@@ -0,0 +1,13 @@
+---
+"@premieroctet/next-admin": major
+---
+
+- Always fetch `id` property from items
```

Comment:
Ideally the identity colum should be an option. We shouldn't force the user to have an id column on the tables.
But this will be an improvement for later :)

Reply:
I've just created an issue for that: https://github.com/premieroctet/next-admin/issues/56